dev-tools/xtask/src/a4x2/a4x2_package.rs

```diff
@@ -0,0 +1,399 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Subcommand: cargo xtask a4x2-package
+
+use super::cmd;
+use anyhow::{Context, Result};
+use camino::{Utf8Path, Utf8PathBuf};
+use clap::Parser;
+use fs_err as fs;
+use sha2::Digest;
+use std::env;
+use std::io::{Read, Write};
+use walkdir::WalkDir;
+use xshell::Shell;
+
+/// This script will be placed in the live tests bundle to run the live tests
+const LIVE_TESTS_EXECUTION_SCRIPT: &str = r#"
+    set -euxo pipefail
```

Comment:
nit: I like to use explicit names here so readers don't have to have these all memorized (e.g., `set -o errexit`).